<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnlockAppLockDialog</class>
 <widget class="QDialog" name="UnlockAppLockDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Разблокировать регистрационную карту</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblLockedCardList">
       <property name="text">
        <string>Заблокированы вашим пользователем:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="lvCode"/>
     </item>
     <item>
      <widget class="QLabel" name="lblInfo">
       <property name="text">
        <string>Введите код пациента:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtCode">
       <property name="maxLength">
        <number>11</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnUnlock">
         <property name="text">
          <string>Разблокировать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnUnlockAll">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Разблокировать всё</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>Закрыть</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
